tests/testthat/test-rvar-print.R

```diff
@@ -1,75 +1,103 @@
 test_that("basic print.rvar works", {
-  skip_on_os("windows")
   x <- rvar(array(1:12, dim = c(2,2,3)))
   x_with_chains <- rvar(array(1:12, dim = c(2,2,3)), nchains = 2)
 
-  expect_output(print(rvar(), color = FALSE),
-"rvar<1>[0] mean ± sd:
-NULL",
-    fixed = TRUE
+  out <- capture.output(print(rvar(), color = FALSE))
+  expect_match(
+    out,
+    regexp = "rvar<1>\\[0\\] mean . sd:",
+    all = FALSE
   )
-
-  expect_output(print(x, color = FALSE),
-"rvar<2>[2,3] mean ± sd:\
-     [,1]         [,2]         [,3]        \
-[1,]  1.5 ± 0.71   5.5 ± 0.71   9.5 ± 0.71 \
-[2,]  3.5 ± 0.71   7.5 ± 0.71  11.5 ± 0.71 ",
-    fixed = TRUE
+  expect_match(
+    out,
+    regexp = "NULL",
+    all = FALSE
   )
 
-  expect_output(print(x_with_chains, color = FALSE),
-"rvar<1,2>[2,3] mean ± sd:\
-     [,1]         [,2]         [,3]        \
-[1,]  1.5 ± 0.71   5.5 ± 0.71   9.5 ± 0.71 \
-[2,]  3.5 ± 0.71   7.5 ± 0.71  11.5 ± 0.71 ",
-    fixed = TRUE
+  out <- capture.output(print(x, color = FALSE))
+  expect_match(
+    out,
+    regexp = "rvar<2>\\[2,3\\] mean . sd:",
+    all = FALSE
+  )
+  expect_match(
+    out,
+    regexp = "     \\[,1\\]         \\[,2\\]         \\[,3\\]        ",
+    all = FALSE
+  )
+  expect_match(
+    out,
+    regexp = "\\[1,\\]  1.5 . 0.71   5.5 . 0.71   9.5 . 0.71 ",
+    all = FALSE
+  )
+  out <- capture.output(print(x_with_chains, color = FALSE))
+  expect_match(
+    out,
+    regexp = "rvar<1,2>\\[2,3\\] mean . sd:",
+    all = FALSE
+  )
+  expect_match(
+    out,
+    regexp = "     \\[,1\\]         \\[,2\\]         \\[,3\\]        ",
+    all = FALSE
+  )
+  expect_match(
+    out,
+    regexp = "\\[1,\\]  1.5 . 0.71   5.5 . 0.71   9.5 . 0.71 ",
+    all = FALSE
+  )
+  expect_match(
+    out,
+    regexp = "\\[2,\\]  3.5 . 0.71   7.5 . 0.71  11.5 . 0.71",
+    all = FALSE
```

Comment:
I had to split these up and use capture.output because with expect_output it kept failing on Windows with "invalid UTF8".